ci-operator/step-registry/baremetalds/assisted/gather/baremetalds-assisted-gather-ref.yaml

```diff
@@ -3,6 +3,7 @@ ref:
   best_effort: true
   optional_on_success: true
   from: assisted-test-infra
+  grace_period: 10m
```

Comment:
Yeah, that is the other corresponsing PR that will be coming. I have changes locally to enforce it but didn't want to put that PR out until this was in as it would make stuff fail.

Reply:
When we announce the feature on the mailing list as well it would be good to link back to this PR so people can see if their config was affected. As it is this can only benefit users with nothing set, so no chance for regression, but they may want to tweak their setting.